cinema/views.py

```diff
@@ -57,7 +63,7 @@ class CinemaHallViewSet(
 ):
     queryset = CinemaHall.objects.all()
     serializer_class = CinemaHallSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The `permission_classes` attribute is correctly set to `IsAdminOrIfAuthenticatedReadOnly`, which aligns with the intended access control for this viewset.